www/sejd00/index.html

```diff
@@ -5,15 +5,63 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>TEST</title>
+    <title>Web firmy</title>
 </head>
 
 <body>
-    <h1>TEST 1</h1>
-    <h2>TEST 2</h2>
-    <h3>vorj00 was here</h3>
-    <h4>Cool!</h4>
-
+    <header>
+        <h1><span>RadioForYou.cz</span></h1>
+        <h2>Jste milovníci nejnovější taneční hudby? U nás naleznete jen to nejlepší.</h2>
+    </header>
+    <nav>
```

Comment:
navigacni prvek by mel byt v headeru, takhle to ale pujde taky, jenom ze to bude na samostatnem radku